sdk/core/azure-core/inc/azure/core/http/body_stream.hpp

```diff
@@ -37,6 +37,20 @@ namespace Azure { namespace Core { namespace Http {
    *@brief Used to read data to/from a service.
    */
   class BodyStream {
+  protected:
+    /**
+     * @brief Read portion of data into a buffer.
+     *
+     * @remark This is the Read implementation that all derived classes need to provide.
+     *
+     * @param conntext #Context so that operation can be canceled.
+     * @param buffer Pointer to a first byte of the byte buffer to read the data into.
```

Comment:
It doesn't have to be the first, right?
```suggestion
     * @param buffer Pointer to a byte buffer to read the data into.
```